bootcamp/tutorials/integration/llamaindex/llamaindex_milvus_full_text_search.ipynb

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.                {"type": "length", "max": 40},  # Custom cap size of a single token
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1516/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>